adyen.php

```diff
@@ -236,9 +276,15 @@ private function removeConfigurationsFromDatabase()
             'ADYEN_CRONJOB_TOKEN'
         );
 
-        $db = Db::getInstance();
-        /** @noinspection SqlWithoutWhere SqlResolve */
-        return $db->execute('DELETE FROM `' . _DB_PREFIX_ . 'configuration` WHERE `name` IN ("' . implode('", "', $adyenConfigurationNames) . '") ');
+        $result = true;
+
+        foreach ($adyenConfigurationNames as $adyenConfigurationName) {
+            if (!Configuration::deleteByName($adyenConfigurationName)) {
+                $result = false;
```

Comment:
should we log this as this should not happen I believe.

Reply:
Yes we can log this as well. Only happens if default prestashop functionality breaks, so something unexpected.